docs/tutorials/migrating-from-docker-compose-to-kubernetes.md

```diff
@@ -0,0 +1,288 @@
+     /
+    ? Provide the minimum number of replicas each service should have
+    Hints:
+    [If the value is 0 pods won't be started by default]
+     2
+    ? Enter the URL of the image registry : 
+    Hints:
+    [You can always change it later by changing the yamls.]
+     quay.io
+    ? Enter the namespace where the new images should be pushed : 
+    Hints:
+    [Ex : myproject]
+     move2kube
+    ? [quay.io] What type of container registry login do you want to use?
+    Hints:
+    [Docker login from config mode, will use the default config from your local machine.]
+     No authentication
+    ? Provide the ingress host domain
+    Hints:
+    [Ingress host domain is part of service URL]
+     localhost
+    ? Provide the TLS secret for ingress
+    Hints:
+    [Leave empty to use http]
+    
+    INFO[0072] Transformer Tekton Done                      
+    INFO[0072] Transformer ComposeGenerator processing 2 artifacts 
+    INFO[0072] Transformer ComposeGenerator Done            
+    INFO[0072] Transformer ClusterSelector processing 2 artifacts 
+    INFO[0072] Transformer ClusterSelector Done             
+    INFO[0072] Transformer Kubernetes processing 2 artifacts 
+    INFO[0072] Transformer Kubernetes Done                  
+    INFO[0072] Transformer ClusterSelector processing 2 artifacts 
+    INFO[0072] Transformer ClusterSelector Done             
+    INFO[0072] Transformer Buildconfig processing 2 artifacts 
+    INFO[0072] Transformer Buildconfig Done                 
+    INFO[0072] Transformer ClusterSelector processing 2 artifacts 
+    INFO[0072] Transformer ClusterSelector Done             
+    INFO[0072] Transformer Knative processing 2 artifacts   
+    INFO[0072] Transformer Knative Done                     
+    INFO[0072] Created 30 pathMappings and 6 artifacts. Total Path Mappings : 30. Total Artifacts : 5. 
+    INFO[0072] Iteration 4 - 6 artifacts to process         
+    INFO[0072] Transformer Parameterizer processing 4 artifacts 
+    INFO[0072] Transformer Parameterizer Done               
+    INFO[0072] Transformer ReadMeGenerator processing 4 artifacts 
+    INFO[0072] Transformer ReadMeGenerator Done             
+    INFO[0072] Plan Transformation done                     
+    INFO[0072] Transformed target artifacts can be found at [/Users/user/Desktop/tutorial/myproject].
+    ```
+    With this we have transformed the Docker Compose app and generated the appropriate Kubernetes YAMLs.
+    Now we can use the scripts that were generated to build the images and push them to our container registry.
+    Then we can deploy the YAMLs to our Kubernetes cluster using `kubectl`. The steps are same as for the [Migration Workflow](https://move2kube.konveyor.io/tutorials/migration-workflow/transform#using-the-output-generated-by-move2kube-transform).
```

Comment:
Hi @HarikrishnanBalagopal, for the docker-compose app, we are not generating the scripts folder. But, as per the [Migration Workflow deployment steps](https://move2kube.konveyor.io/tutorials/migration-workflow/transform#using-the-output-generated-by-move2kube-transform), it requires running the scripts inside the `scripts` folder and the users may get confused as there won't be any `scripts` folder inside the `myproject` folder. So, do you think it would be good to show the deployment step (kubectl apply) for the docker-compose app in the same tutorial, may be along with a screenshot of the deployed app on the cluster?

Reply:
@HarikrishnanBalagopal Proabably we can replace the first line with, `Build the docker images as was done in docker compose flow and push them to the registry or you can also automate the flow using the tekton pipeline found in deploy/cicd directory.